libsigner/src/http.rs

```diff
@@ -138,7 +138,7 @@ pub fn decode_http_response(payload: &[u8]) -> Result<(HashMap<String, String>,
     let mut headers_buf = [httparse::EMPTY_HEADER; MAX_HTTP_HEADERS];
     let mut resp = httparse::Response::new(&mut headers_buf);
 
-    // consume respuest
+    // consume request
```

Comment:
I think this should be "response" instead.

Reply:
Yes, it's `response`.